safe-global/safe-wallet-web
# Test: Add "sign action" to the replace signer e2e test

## What it solves
Adds the happy path for "replace signer" test

## How this PR fixes it
Adds a new happy path file. 
In a new safe it replaces the existing owner, signgs the tx and check the tx is in the queue list. The tx is never executed
Since there is a new folder for happy path I also moved the "remove signer hp" test from the Remove owner test. The test has not changed.
I tried to rename "owner" for "signer" in every place I could because that's something we have to do for all test now

## How to test it
Run "replace signer hp" file and check it passes
Run "remove signer hp" and check that is passes (the test has not changed from the replace owner test, but still make sure it works)

## Checklist
